LinearAlgebraForCAP/gap/precompiled_categories/MatrixCategoryPrecompiled.gi

```diff
@@ -1867,11 +1867,11 @@ function ( cat_1, arg2_1, arg3_1 )
     local deduped_1_1, deduped_2_1;
     deduped_2_1 := UnderlyingMatrix( arg3_1 );
     deduped_1_1 := UnderlyingMatrix( arg2_1 );
-    return IsZero( deduped_1_1 * SyzygiesOfColumns( deduped_2_1 ) ) and IsZero( deduped_2_1 * SyzygiesOfColumns( deduped_1_1 ) );
+    return IsZero( DecideZeroRows( deduped_1_1, deduped_2_1 ) ) and IsZero( DecideZeroRows( deduped_2_1, deduped_1_1 ) );
```

Comment:
For the record: I can think of a context in which this change would pose a regression. This would require an implementation of `SyzygiesOfColumns`, which is cheaper than `BasisOfRowModule` which is called by `DecideZeroRows`.

Reply:
For the record 2: We have decided to not tackle this point for now, because

1. When using Singular, `BasisOfRowModule` is cheaper than `SyzygiesOfColumns`,
2. probably there are many more of such cases in `LinearAlgebraForCAP`, so one should go over the whole code at once,
3. if we make the primitive installation of `IsLiftable` more expensive, than it will be derived from the homomorphism structure leading to an infinite recursion, because the derivation mechanism currently does not take the weights in the range category into account.